test/core/rules/config_test.py

```diff
@@ -26,10 +26,14 @@ class Rule_T001(BaseRule):
 
     def _eval(self, context):
         """Stars make newlines."""
-        if context.segment.is_type("star"):
+        if context.segment.is_type("whitespace"):
             return LintResult(
                 anchor=context.segment,
-                fixes=[LintFix.create_before(context.segment, [NewlineSegment()])],
+                fixes=[
+                    LintFix.replace(
+                        context.segment, [WhitespaceSegment(context.segment.raw + " ")]
+                    )
+                ],
```

Comment:
I had to update this dummy rule, because the old version corrupted the parse tree, which we now catch and report.